oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,103 @@
+Deploy Ubuntu Pro container image to Pro Kubernetes cluster
+===========================================================
+
+You can easily deploy your Ubuntu Pro container images to Ubuntu Pro Kubernetes clusters.
+The Kubernetes cluster needs to be on Ubuntu Pro nodes. Read more on the
+`Ubuntu Pro <https://ubuntu.com/pro>`_ website.
+
+
+Prerequisites
+-------------
+
+You will need --
+
+- an Ubuntu Pro container image on a private registry. See how to build one
+  :doc:`here <build-ubuntu-pro-container-image>`.
+  The image should be kept private to avoid sharing your Ubuntu Pro subscription with
+  unwanted users since the image is linked to your Ubuntu Pro subscription.
+  
+  You can publish the image to your private registry, with:
+  
+  ::
+
+		docker push <private-pro-image-name>
+
+- a Kubernetes cluster with a Ubuntu Pro subscription. See the cloud-specific guides to
+  learn how to deploy it for :doc:`EKS </aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst>`
+  and :doc:`GCE </google/gce/gce-how-to/deploy-kubernetes-with-ubuntu-pro>`.
+
+
+Create a Secret for Private Registry
+------------------------------------
+
+If you published your image to a private registry, you will need to create a
+`secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_ in Kubernetes.
+`See here <https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/>`_
+to learn about pulling images from private registries.
+
+You can use the following example command to create a secret named ``regcred`` for
+`Docker Hub <https://hub.docker.com>`_. Use ``https://index.docker.io/v1/``
+as ``<your-registry-server>``.
+
```

Comment:
Are we suggesting the use of ``https://index.docker.io/v1/`` for some specific reason? If so, mention that reason. This statement sounds like a mandatory requirement.

Reply:
Yes, I understand. Adding the 'For this' at the beginning of the statement just makes it a little clearer that the specific URL is needed because we are giving Dockerhub as an example.

Reply:
Ah okay, thanks. Let me add another commit. :v

Reply:
:) thanks!

Reply:
Done!